src/docs/guides/16-how-to-use-ci-with-ftw/index.md

```diff
@@ -41,3 +41,15 @@ This command ensures that the build passes.
 ### Tests: `yarn run test-ci`
 
 This command runs the tests.
+
+### Security audit: `yarn run audit`
+
+This command runs the security audit using `yarn audit --json` and
+checks returned JSON against vulnerability exceptions defined in
+`.nsprc` file at the project root. The audit checks for installed
```

Comment:
Make sure to update this doc if you make changes to the FTW PR.

Also, change the article updated date.

Reply:
Updated and updated